charts/postgres-operator/templates/operator-service-account-rbac.yaml

```diff
@@ -0,0 +1,290 @@
+apiVersion: v1
```

Comment:
directly copied from zalando. I'm unsure if our users will want templating in this file to tweak values.